src/robot/ConcreteRobot.cpp

```diff
@@ -245,16 +264,34 @@ TrajectoryPtr ConcreteRobot::planToConfiguration(
     const CollisionFreePtr& collisionFree,
     double timelimit)
 {
+  DART_UNUSED(timelimit);
+
+  auto robot = metaSkeleton->getBodyNode(0)->getSkeleton();
```

Comment:
This PR seems like a good time to address https://github.com/personalrobotics/aikido/issues/447.